src/components/AboutContainer/AboutSideBar/AboutSideBar.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import useStyles from './AboutSideBar.styles';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
```

Comment:
Nothing super constructive. Just prefer to use their icon API
`import { ChevronRight, ChevronLeft } from '@material-ui/icons';`

Reply:
Fixed in b57f2c7f4a31b6e2bfb46523d020db6754cdb21e